lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme_handlers.cpp

```diff
@@ -3771,10 +3773,9 @@ status_code_e s1ap_handle_new_association(s1ap_state_t* state,
     }
     enb_association->sctp_assoc_id = sctp_new_peer_p->assoc_id;
     enb_association->enb_id = 0xFFFFFFFF;  // home or macro eNB is 28 or 20bits.
-    hashtable_rc_t hash_rc = hashtable_ts_insert(
-        &state->enbs, (const hash_key_t)enb_association->sctp_assoc_id,
-        (void*)enb_association);
-    if (HASH_TABLE_OK != hash_rc) {
+    magma::proto_map_rc_t rc =
+        state->enbs.insert(enb_association->sctp_assoc_id, enb_association);
+    if (magma::PROTO_MAP_OK != rc) {
```

Comment:
Keep the variable on left hand side of comaprision
```suggestion
    if (rc != magma::PROTO_MAP_OK ) {
```